recipes/opentelemetry-propagator-b3/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "opentelemetry-propagator-b3" %}
+{% set version = "0.17b0" %}
```

Comment:
This is a beta release, right? Is there a stable version that we can add to the main channel? If not we need to follow: https://github.com/conda-forge/cfep/blob/master/cfep-05.md

Reply:
Hi. There's no stable version, just like dependency https://anaconda.org/conda-forge/opentelemetry-api .

Reply:
yeah none of the the opentelemetry things have stable releases yet.  It seems to the their zerover phase to append a b0 to everything

Reply:
@ocefpaf This falls under the same policy as black so we luckily don't need to do something special

Reply:
Yep. Thanks @mariusvniekerk! (Just wanted to check before other core member start screaming at me ;-p)